### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by sserunkumajoel4** [community/community#75456] [Other Feature Feedback, Questions, & Ideas]
why am seeing DNS check in progress for 2days
and even published website if visited it is ssid not be secure

**Comment by elderguardian** [comment]
This sounds like you entered a custom domain into gh pages which does not have the required dns settings. 
Create a CNAME record like this in your dns settings: your-name.github.io


you will be able to force https afterwards which will secure your site

**Comment by OfficialChrisTucker** [comment]
If you are seeing a "DNS check in progress" message for 2 days, it could indicate an issue with the DNS configuration for your website. The DNS (Domain Name System) is responsible for translating domain names (e.g., www.example.com) into IP addresses, allowing your website to be accessed by users.

**Comment by OfficialChrisTucker** [comment]
Here are a few possible reasons why you might be experiencing this issue:

 DNS propagation: When you make changes to your DNS settings (such as updating the IP address or adding new records), it can take some time for these changes to propagate across the internet. This propagation period can vary, ranging from a few minutes to 48 hours. During this time, you may see the "DNS check in progress" message.

 ncorrect DNS configuration: A misconfigured DNS record can also result in the "DNS check in progress" message. Ensure that you have set up the correct DNS records (such as A, CNAME, or MX records) with the appropriate values.

Regarding the "not secure" message when visiting your website, it typically indicates that your website is not using an SSL certificate, or the SSL certificate is not configured correctly. SSL (Secure Sockets Layer) certificates encrypt the connection between a visitor's browser and your web server, ensuring that data transmitted is secure.

To resolve the "not secure" issue, you can follow these steps:

Obtain an SSL certificate: Purchase an SSL certificate from a trusted certificate authority (CA) or use a free certificate provider like Let's Encrypt.

 Install the SSL certificate: Follow the instructions provided by your hosting provider to install the SSL certificate on your web server. This process may involve generating a certificate signing request (CSR) and applying the certificate to your domain.

Update website URLs: Once the SSL certificate is installed, ensure that all URLs used on your website (including internal links, assets, and external resources) are updated to use the "https://" protocol instead of "http://".

By addressing these issues, you should be able to resolve the "DNS check in progress" message and secure your website with an SSL certificate. If you encounter any further difficulties, it may be beneficial to consult with your hosting provider or seek assistance from a web developer to ensure a smooth resolution.
